client/state/action-types.js

```diff
@@ -182,6 +182,8 @@ export const NOTICE_REMOVE = 'NOTICE_REMOVE';
 export const OLARK_READY = 'OLARK_READY';
 export const OLARK_REQUEST = 'OLARK_REQUEST';
 export const OLARK_TIMEOUT = 'OLARK_TIMEOUT';
+export const OLARK_OPERATORS_AVAILABLE = 'OLARK_OPERATORS_AVAILABLE';
+export const OLARK_OPERATORS_AWAY = 'OLARK_OPERATORS_AWAY';
```

Comment:
I did and I wanted to go with the convention used for the `OLARK_READY` action. Then I figured either way works just as well as the other and I like the multi cased reducer vs the single with an `if`

```js
	switch ( action.type ) {
		case OLARK_OPERATORS_AVAILABLE:
			return OPERATOR_STATUS_AVAILABLE;
		case OLARK_OPERATORS_AWAY:
			return OPERATOR_STATUS_AWAY;
	}
	return state;
```

vs 

```js
	switch ( action.type ) {
		case OLARK_OPERATORS_STATUS_SET:
			return action.isAvailable ? OPERATOR_STATUS_AVAILABLE : OPERATOR_STATUS_AWAY;
	}
	return state;
```